### PR TITLE
Switch dotnet-bundle to BuildBundlerMinifier

### PR DESCRIPTION
**Pull request by natemcmaster**
Most recent update of BuildBundlerMinifier chains the bundle step in automatically via MSBuild.

cref https://github.com/madskristensen/BundlerMinifier/issues/230

### CODE REVIEW

File: samples/ChatSample/ChatSample.csproj [hunk 1]
```diff
@@ -15,6 +15,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="BuildBundlerMinifier" Version="2.4.337" />
```
Comment (moozzyk) [inline samples/ChatSample/ChatSample.csproj]: How it is being invoked?

File: samples/ChatSample/ChatSample.csproj [hunk 2]
```diff
@@ -53,7 +53,6 @@
   </Target>
 
   <ItemGroup>
-    <DotNetCliToolReference Include="BundlerMinifier.Core" Version="2.2.301" />
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="$(AspNetCoreVersion)" />
     <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.1-*" />
```
Comment (moozzyk) [inline samples/ChatSample/ChatSample.csproj]: Is there a $(VersionVariable) we could use here?

File: samples/ChatSample/ChatSample.csproj (hunk 1 above)
Comment (natemcmaster) [inline samples/ChatSample/ChatSample.csproj]: This package carries an [MSBuild target](https://github.com/madskristensen/BundlerMinifier/blob/master/src/BundlerMinifier/build/BuildBundlerMinifier.targets) that chains in automatically to "Build" and "Clean".

File: samples/ChatSample/ChatSample.csproj (hunk 2 above)
Comment (natemcmaster) [inline samples/ChatSample/ChatSample.csproj]: We could make one but we should bump these tools to 1.2.0-* like everything else.

Comment (moozzyk) [inline samples/ChatSample/ChatSample.csproj]: We should but this is not this change.

File: samples/ChatSample/ChatSample.csproj (hunk 1 above)
Comment (moozzyk) [inline samples/ChatSample/ChatSample.csproj]: I see. Thanks!

**Comment by moozzyk** [comment]
:ship::it:

**Pull request merged**